cloud/src/meta-store/keys.h

```diff
@@ -73,12 +75,41 @@
 // 0x02 "system" "meta-service" "registry"                                                 -> MetaServiceRegistryPB
 // 0x02 "system" "meta-service" "arn_info"                                                 -> RamUserPB
 // 0x02 "system" "meta-service" "encryption_key_info"                                      -> EncryptionKeyInfoPB
+//
+// 0x03 "version" ${instance_id} "partition" ${partition_id} ${timestamp}   -> VersionPB
+// 0x03 "version" ${instance_id} "table" ${table_id} ${timestamp}           -> ${empty_value}
+//
+// 0x03 "index" ${instance_id} "partition" ${partition_id}                                                  -> PartitionIndexPB
+// 0x03 "index" ${instance_id} "partition_inverted" ${db_id} ${table_id} ${index_id} ${partition}           -> ${empty_value}
+// 0x03 "index" ${instance_id} "tablet" ${tablet_id}                                                        -> TabletIndexPB
+// 0x03 "index" ${instance_id} "tablet_inverted" ${db_id} ${table_id} ${index_id} ${partition} ${tablet}    -> ${empty_value}
+// 0x03 "index" ${instance_id} "index" ${index_id}                                                          -> IndexIndexPB
+// 0x03 "index" ${instance_id} "index_inverted" ${db_id} ${table_id} ${index_id}                            -> ${empty_value}
+//
+// 0x03 "stats" ${instance_id} "tablet_load" ${tablet_id} ${timestamp} -> TabletStatsPB
+// 0x03 "stats" ${instance_id} "tablet_compact" ${tablet_id} ${timestamp} -> TabletStatsPB
+//
+// 0x03 "meta" ${instance_id} "partition" ${partition_id} ${timestamp}              -> ${empty_value}
+// 0x03 "meta" ${instance_id} "index" ${index_id} ${timestamp}                      -> ${empty_value}
+// 0x03 "meta" ${instance_id} "tablet" ${tablet_id} ${timestamp}                    -> TabletMetaPB
+// 0x03 "meta" ${instance_id} "schema" ${index_id} ${schema_version}                -> TabletSchemaPB
+// 0x03 "meta" ${instance_id} "rowset_load" ${tablet_id} ${version} ${timestamp}    -> RowsetMetaPB
+// 0x03 "meta" ${instance_id} "rowset_compact" ${tablet_id} ${version} ${timestamp} -> RowsetMetaPB
+//
+// 0x03 "data" ${instance_id} "rowset" ${tablet_id} ${rowset_id} => int64
```

Comment:
what is this for?

Reply:
data path reference count

Reply:
suggested scheme
// 0x03 "data" ${instance_id} "rowset_ref_cnt" ${tablet_id} ${rowset_id} => int64